app/javascript/packages/analytics/index.ts

```diff
@@ -1,11 +1,6 @@
-import type { noticeError } from 'newrelic';
 import { getConfigValue } from '@18f/identity-config';
 
-type NewRelicAgent = { noticeError: typeof noticeError };
-
-interface NewRelicGlobals {
-  newrelic?: NewRelicAgent;
-}
+export { default as isTrackableErrorEvent } from './is-trackable-error-event';
```

Comment:
Follow-up task: Would like to move `trackEvent` and `trackError` to dedicated files `track-event.ts` and `track-error.ts` so this `index.ts` can actually be an index.